plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -45,27 +63,12 @@ class SlurmQueueConf:
 
 
 @dataclass
-class LocalQueueConf:
-    # local executor mocks the behavior of slurm locally
-
-    # maximum time for the job in minutes
-    timeout_min: int = 60
-    # number of gpus to use on each node
-    gpus_per_node: int = 1
-    # number of tasks to spawn on each node (only one node available in local executor)
-    tasks_per_node: int = 1
-
-
-@dataclass
-class AutoQueueConf:
-    # auto executor automatically identifies and uses available cluster
-    # Currently this is only slurm, but local executor can be manually forced
-    # instead.
-    # Most parameters are shared between clusters, some can be cluster specific
+class SubmititConf(BaseSubmititConf):
+    folder: str = "${hydra.sweep.dir}/.${hydra.launcher.params.queue}"
 
-    # cluster to use (currently either "slurm" or "local" are supported,
+    # executor to use (currently either "slurm" or "local" are supported,
     # None defaults to an available cluster)
-    cluster: str = "slurm"
+    executor: ExecutorName = ExecutorName.slurm
 
     # maximum time for the job in minutes
     timeout_min: int = 60
```

Comment:
by default all the values here are shared for all clusters/executors. But they can always be bypassed